docs/packages/pkg/EnumGroup.rst

```diff
@@ -0,0 +1,20 @@
+.. spelling::
+
+    EnumGroup
+
+.. index:: unsorted ; EnumGroup
+
+.. _pkg.EnumGroup:
+
+EnumGroup
+==========
+
+-  `Official <https://github.com/Person-93/EnumGroup>`__
+-  `Hunterized <https://github.com/hunter-packages/EnumGroup>`__
```

Comment:
There is no hunterized repository, please remove.

Reply:
Done